Tests/UI/New UI/BillingForm/ViewModel/BillingFormViewModelTests.swift

```diff
@@ -30,7 +30,7 @@ class BillingFormViewModelTests: XCTestCase {
         textField.text = text
 
         viewModel.validate(text: textField.text, cellStyle: expectedType, row: tag)
-        XCTAssertEqual( viewModel.errorFlagOfCellType[expectedType.index], false)
+        XCTAssertEqual(viewModel.errorFlagOfCellType[expectedType.index], true)
```

Comment:
Just mentioning but please try to use `XCTAssertTrue` and `XCTAssertFalse` instead of equal between booleans.